lib/grape-active_model_serializers/error_formatter.rb

```diff
@@ -0,0 +1,54 @@
+require 'multi_json'
+module Grape
+  module ErrorFormatter
+    module ActiveModelSerializers
+      extend Base
+
+      class << self
+        # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
Get rid of of the rubocop stuff here and do `rubocop -a ; rubocop --auto-gen-config`.

Reply:
```rubocop -a ; rubocop --auto-gen-config
Inspecting 31 files
...C...........................

Offenses:

lib/grape-active_model_serializers/error_formatter.rb:8:9: C: Cyclomatic complexity for call is too high. [7/6]
        def call(message, backtrace, options = {}, env = nil, original_exception = nil)
        ^^^
lib/grape-active_model_serializers/error_formatter.rb:8:81: C: Line is too long. [87/80]
        def call(message, backtrace, options = {}, env = nil, original_exception = nil)
                                                                                ^^^^^^^

31 files inspected, 2 offenses detected
Inspecting 31 files
..CCCCC.C.......C..............

Offenses:

lib/grape-active_model_serializers/endpoint_extension.rb:8:3: C: Missing top-level module documentation comment.
  module EndpointExtension
  ^^^^^^
lib/grape-active_model_serializers/error_formatter.rb:4:5: C: Missing top-level module documentation comment.
    module ActiveModelSerializers
    ^^^^^^
lib/grape-active_model_serializers/error_formatter.rb:8:9: C: Cyclomatic complexity for call is too high. [7/6]
        def call(message, backtrace, options = {}, env = nil, original_exception = nil)
        ^^^
lib/grape-active_model_serializers/error_formatter.rb:8:81: C: Line is too long. [87/80]
        def call(message, backtrace, options = {}, env = nil, original_exception = nil)
                                                                                ^^^^^^^
lib/grape-active_model_serializers/formatter.rb:3:5: C: Missing top-level module documentation comment.
    module ActiveModelSerializers
    ^^^^^^
lib/grape-active_model_serializers/options_builder.rb:3:5: C: Missing top-level class documentation comment.
    class OptionsBuilder
    ^^^^^
lib/grape-active_model_serializers/serializer_resolver.rb:3:5: C: Missing top-level class documentation comment.
    class SerializerResolver
    ^^^^^
lib/grape-active_model_serializers.rb:1:1: C: The name of this source file (grape-active_model_serializers.rb) should use snake_case.
require 'active_model_serializers'
^
spec/grape-active_model_serializers_spec.rb:1:1: C: The name of this source file (grape-active_model_serializers_spec.rb) should use snake_case.
require 'spec_helper'
^

31 files inspected, 9 offenses detected
Created .rubocop_todo.yml.
Run `rubocop --config .rubocop_todo.yml`, or add `inherit_from: .rubocop_todo.yml` in a .rubocop.yml file.```

Reply:
Yes, and now Rubocop will run clean.